docker-compose.dev.yaml

```diff
@@ -0,0 +1,34 @@
+services:
+  mongodb:
+    image: bitnami/mongodb:4.2
+    volumes:
+      - mongodb_data:/bitnami
+  backend:
+    build: 
+      context: .
+      target: dev
+    depends_on:
+      - mongodb
+    ports:
+      - 3000:3000
+    volumes:
+      - .:/home/node/app
+      - /home/node/app/node_modules
+    environment:
+      MONGODB_URI: mongodb://mongodb:27017/scicat
+      EXPRESS_SESSION_SECRET: a_scicat_secret
+      JWT_SECRET: a_scicat_secret
+      PORT: 3000
+      HTTP_MAX_REDIRECTS: 5
+      HTTP_TIMEOUT: 5000
+      JWT_EXPIRES_IN: 3600
+      SITE: SAMPLE-SITE
+      PID_PREFIX: PID.SAMPLE.PREFIX
+      DOI_PREFIX: DOI.SAMPLE.PREFIX
+      METADATA_KEYS_RETURN_LIMIT: 100
+      METADATA_PARENT_INSTANCES_RETURN_LIMIT: 100
+    command: /bin/sh -c "while true; do sleep 600; done"
```

Comment:
why is it necessary to run this sleep command?

Reply:
the idea of this docker-compose is to have an env which you can attach to do your local dev (that's also why the code is mounted). So the sleep is to keep the container busy so that you can attach to it